site/e2e/tests/logout.spec.ts

```diff
@@ -0,0 +1,13 @@
+import { test, expect } from "@playwright/test"
+import { getStatePath } from "../helpers"
+
+test.use({ storageState: getStatePath("authState") })
+
+test("signing out redirects to login page", async ({ page, baseURL }) => {
+  await page.goto(`${baseURL}/`, { waitUntil: "networkidle" })
+
+  await page.getByTestId("user-dropdown-trigger").click()
+  await page.getByRole("menuitem", { name: "Sign Out" }).click()
+
+  await expect(page.getByRole("heading", { name: "Sign in to Coder" })).toBeVisible()
```

Comment:
Can we check if the URL has no query parameters after logout? I see that sometimes when doing logout, the "redirectTo" is getting populated. PS: Idk if the query param is exactly this one but we have a param to redirect a user to a page after login.